consensus/hotstuff/cruisectl/README.md

```diff
@@ -180,11 +181,11 @@ In particular systematic observation bias are a problem, as it leads to a diverg
 ```math
 \eqalign{
 \textnormal{initialization: }\quad \bar{\mathcal{I}} :&= 0 \\
-\textnormal{update with instantaneous error\ } e[v]:\quad \bar{\mathcal{I}}[v] &= e[v] + (1-\beta)\cdot\bar{\mathcal{I}}[v-1]
+\textnormal{update with instantaneous error\ } e[v]:\quad \bar{\mathcal{I}}[v] &= e[v] + (1-\lambda)\cdot\bar{\mathcal{I}}[v-1]
```

Comment:
renamed $\beta$ to $lambda$ here, as I accidentally re-used $\beta$ for a different quantity. With this change, the nomenclature should be unambiguous.